Merge conflict arises if we try to merge one branch to another where certain lines of codes in one branch is committed
in one branch and the same lines of code in the other branch is also committed.


a = 1
b = 2
print("a*b")